daniddelrio/urc-seagrass
# Popup component on click of polygon

Covers these tickets:
1. Create base Popup component
2. Highlight polygons based on GeoJSON data
3. Allow created Popup component to appear on click of polygon

To test:
1. cd into the client folder and run the server by doing `npm start`.
2. The map should appear on the index page. Try hovering over a highlighted area and hovering out to see if the hover effect works.
3. Click on a highlighted area to see the Popup component on click, along with the custom data found in the JSON file.

Basis of acceptance
1. Highlighted areas are seen on the map
2. Popup component appears on click of polygon
3. Popup component looks good on both desktop and mobile.

NOTE:
- The polygons were made randomly on [geojson.io](http://geojson.io/) and are not reflective of the actual areas to be shown. Same goes for the data that were shown in the popups.
- The different years are yet to be implemented.
